connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -372,14 +379,21 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
   // checks for cancellation and throws if it's the case
   await Context.current().sleep(1);
 
-  if (pageCount > 0) {
-    await syncSucceeded(connector.id);
-  } else {
+  if (totalExtracted <= 0) {
+    await syncFailed(connector.id, "webcrawling_error_empty_content");
+  } else if (upsertedPageCount <= 0) {
+    // this means that every page was too big to be upserted
+    await syncFailed(connector.id, "webcrawling_error_content_too_large");
+  } else if (!isNotBlocked) {
+    await syncFailed(connector.id, "webcrawling_error_blocked");
```

Comment:
wouldn't do that here because if only 1 page is 403, we mark the whole site as failed
i'd go for > 10% pages blocked, and error message accordingly
(also cc @albandum wdyt?)

Reply:
Alternatively, all pages or a very low ratio; maybe tbd on what actually happens then

Reply:
maybe the other way round => if we can crawl more than 10% pages we're good

Reply:
maybe even start with only 1%

Reply:
Yeah def we don't want to crash as soon as we fail 1 page - agree with the 1%